setup.py

```diff
@@ -31,30 +35,8 @@
     keywords = "Earley LALR parser parsing ast",
     url = "https://github.com/erezsh/lark",
     download_url = "https://github.com/erezsh/lark/tarball/master",
-    long_description='''
-Lark is a modern general-purpose parsing library for Python.
-
-With Lark, you can parse any context-free grammar, efficiently, with very little code.
-
-Main Features:
- - Builds a parse-tree (AST) automagically, based on the structure of the grammar
- - Earley parser
-    - Can parse all context-free grammars
-    - Full support for ambiguous grammars
- - LALR(1) parser
-    - Fast and light, competitive with PLY
-    - Can generate a stand-alone parser
- - CYK parser, for highly ambiguous grammars
- - EBNF grammar
- - Unicode fully supported
- - Python 2 & 3 compatible
- - Automatic line & column tracking
- - Standard library of terminals (strings, numbers, names, etc.)
- - Import grammars from Nearley.js
- - Extensive test suite
- - And much more!
-''',
-
+    long_description=README,
```

Comment:
To do this, `README.md` should also be changed, since there are a few things that will break when not used on github.